static/src/javascripts/bootstraps/enhanced/newsletters.js

```diff
@@ -44,6 +44,15 @@ const addSubscriptionMessage = (buttonEl: HTMLButtonElement): void => {
     });
 };
 
+const modifyDataLinkName = (modifier: string) => (el: HTMLButtonElement) : void => {
+    const firstStageName = el.getAttribute('data-link-name') || ""
```

Comment:
```suggestion
    const firstStageName = el.getAttribute('data-link-name') || "undefined"
```
This is currently set to an empty string for if it's not present, but could potentially be modified to use undefined if this better describes the situation